actions/health-and-version-check/src/health-check.ts

```diff
@@ -37,15 +40,15 @@ async function checkHealth(
     const attempts = Math.round(numberOfSeconds / 10);
     let result = false;
 
+    info(`Checking ${webAppName} health status`);
+    info(`Url: ${url}`);
+
     for (let index = 1; index < attempts; index++) {
-        info(`Checking ${webAppName} health status`);
-        info(`Url: ${url}`);
+        await new SleepTimer().sleep(10000);
```

Comment:
We have https://github.com/ritterim/public-github-actions/blob/master/actions/health-and-version-check/src/health-check.ts#L12 that should add a wait before the call starts. This will add another wait for each call loop.